webapp/tests/karma/ts/modules/messages/messages-content.component.spec.ts

```diff
@@ -245,5 +246,51 @@ describe('MessagesContentComponent', () => {
       expect(removeMessageFromSelectedConversationSpy.args[0]).to.deep.equal(['message2']);
     });
   });
+
+  describe.only('New message section', () => {
+    it('should display the "new message" section in a conversation with a known recipient', async () => {
+      component.selectedConversation = {
+        'id': '7aeaa4bc-ae5e-4964-8771-697209d2c6dd',
+        'messages': [
+          { 'id': '68e4ae1a-bb6c-4561-98e7-3f7cdfd00486' }
+        ],
+        'contact': {
+          'doc': { 'name': 'Groot' }
+        }
+      };
+
+      fixture.detectChanges();
+      await fixture.whenStable();
+
+      const newMessageInput = fixture
+        .debugElement
+        .query(By.css('#message-footer textarea[name="message"]'))
+        ?.nativeElement;
+      expect(newMessageInput).to.not.be.undefined;
+    });
+
+    it('should hide the "new message" section in a conversation with an unknown recipient', async () => {
+      component.selectedConversation = {
+        'id': '52c1de4a-33dd-4d22-9916-24e7c77451f5',
+        'messages': [
+          { 'id': '40b0678a-0f16-47cb-8591-fb3eab3b81d2' }
+        ],
+        'contact': {
+          'doc': {
+            'name': ''
+          }
+        }
+      };
+
+      fixture.detectChanges();
+      await fixture.whenStable();
+
+      const unknownContactMessage = fixture
+        .debugElement
+        .query(By.css('#unknown-contact-error-message'))
```

Comment:
Same as above, please add an assertion about the text area not being displayed. Right now, if we regress and actually display the textarea for unknown contacts, this test would not fail.